dffml/record.py

```diff
@@ -141,31 +143,75 @@ def __repr__(self):
         return str(self.dict())
 
     def __str__(self):
-        header = self.key
+        try:
+            width = int(3 * os.get_terminal_size().columns / 4)
+        except OSError as e:
+            width = 70
+        header = "\n\tKey:\t" + self.key
+        feature_header = "Record Features".center(int(width)).rstrip()
+        divider = "+" + "-" * (width) + "+"
+
         if len(self.extra.keys()):
-            header += " " + str(self.extra)
+            header += "\n\t" + str(self.extra)
 
         return "\n".join(
             [header]
+            + [feature_header]
+            + [divider]
             + [
-                ("%-30s%s" % (feature, str(results)))
+                (
```

Comment:
This "building a table from a dict" should be it's own function. Maybe it should go somewhere like `dffml.util.display`? `display` may not be the best word choice, if you think of something else go for it.